src/test/unit/specific_feature_tests/dialogs.js

```diff
@@ -173,266 +144,235 @@ function getDialogDocumentTitle(dialogName) {
     return TITLES[dialogName];
 }
 
-test("Link dialog doesn't open when no selection; by toolbar button click",
```

Comment:
Removed `by toolbar button click` from the names of all of these tests because we now test not only by toolbar button click, but also by key combo (in the case of the link dialog).